sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -216,7 +216,7 @@ def __init__(
           'InfluxDB')
     self.filters = filters
 
-  def publish_metrics(self, result, extra_metrics: dict):
+  def publish_metrics(self, result, extra_metrics: dict = None):
```

Comment:
you may need to tag the type as `Optional[Dict]]` so that this does not give an error.

Reply:
Done 👍